src/main/java/se/sundsvall/document/api/DocumentResource.java

```diff
@@ -62,7 +63,7 @@
 
 @RestController
 @Validated
-@RequestMapping("/documents")
+@RequestMapping("{municipalityId}/documents")
```

Comment:
A leading slash could be added here. It works both with and without, but if you don't add any slash, your code will depend on the following line: 
https://github.com/ndimiduk/spring-framework/blob/64e0b1dc25773bbf344a8a15c35ee6fb8d06cf2a/org.springframework.web.servlet/src/main/java/org/springframework/web/servlet/mvc/annotation/DefaultAnnotationHandlerMapping.java#L120